script/check_enforced_eslint_rules

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+SCRIPT_NAME=$(basename "$0")
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [--help]
+
+Check all ESLint rules enforced in this repository so far.
+EOH
+)
+
+if [[ $# -ne 0 && $1 =~ -*h(elp)? ]]
+then
+  echo "$HELP"
+  exit 0 # EX_OK
+elif [[ $# -ne 0 ]]
+then
+  echo >&2 "$SCRIPT_NAME: unrecognized argument: $1"
+  echo >&2 "Try '$SCRIPT_NAME help' for usage."
+  exit 64 # EX_USAGE
+fi
+
+cd "$(dirname "${BASH_SOURCE[0]}")/../" || exit 66 # EX_NOINPUT
+
+./script/check_eslint_rule \
+  'indent: [warn, 2, {CallExpression: {arguments: first}, SwitchCase: 1, ignoredNodes: ["JSXElement", "ArrowFunctionExpression"]}]' \
+  --rule 'react/jsx-indent: [error, 2]' \
+  --rule 'react/jsx-indent-props: [error, 2]' \
+  --max-warnings 0 \
+  .
```

Comment:
Thanks! This will be very useful to have.

I wish we didn't have to duplicate all the rule configuration here. There've already been several times where I forgot to update configuration in check_eslint_rule. :(

What do you think about just commenting out all the rules in .eslintrc.yaml we haven't fixed yet? Then we can uncomment them as we verify they're fixed. The CI command becomes as simple as running `./node_modules/.bin/eslint`.

Reply:
This new script just checks the bare minimum to pass CI tests, but I still use the full `\.eslintrc` when creating new files.

Reply:
Right, but eventually it'll duplicate a lot of rules (otherwise we'd just fix them all at once).

What about having two configs, `.eslintrc.yaml` and `.eslintrc.unfixed.yaml`? You can specify the additional config like `-c .eslintrc.unfixed.yaml` and eslint will merge them.

Reply:
Adopted it and added a wrapper script for checking extra rules by hand.

Reply:
Would this two-file setup also allow us to have the Github auto-check check PRs for fixed rules?

Reply:
Yes but CircleCI does the job, see commit https://github.com/metabrainz/musicbrainz-server/pull/1696/commits/be6616aea2ff1c11033da0712b4d68258477d9c2.